test/e2e/app/app.go

```diff
@@ -27,6 +27,23 @@ type Application struct {
 	restoreChunks   [][]byte
 }
 
+type Config struct {
+	Dir              string
+	SnapshotInterval uint64
+	RetainBlocks     uint64
+	KeyType          string
+	PersistInterval  uint64
```

Comment:
What are the units here? Is this number of blocks prior to the current height?

Reply:
Yup. I'll copy over some of the explanations